float64 altitude_target
float64 timeout
---
bool success